com.pizi.scrcpygui.yml

```diff
@@ -0,0 +1,68 @@
+app-id: com.pizi.scrcpygui
+runtime: org.gnome.Platform 
+runtime-version: '47' 
+sdk: org.gnome.Sdk 
+command: scrcpygui
+build-options:
+  prepend-path: /app/scrcpygui
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=all
+  - --socket=pulseaudio
+  - --filesystem=xdg-data
+  - --filesystem=home
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.Avahi.*
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+ 
+  - name: flutter-scrcpygui
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    depends:
+      - libayatana-appindicator
+    sources:
+      - type: file
+        url: https://github.com/pizi-0/flutter-scrcpygui/releases/download/1.3.50/scrcpygui-1.3.50-linux.zip
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source